Dockerfile.test.db

```diff
@@ -17,9 +17,14 @@ RUN echo "Use LorisTest;" | cat - \
     0000-00-02-Menus.sql \
     0000-00-03-ConfigTables.sql \
     0000-00-04-Help.sql \
-    0000-01-00-TestUser.sql \
+    test_instrument.sql \
     radiology_review.sql > /docker-entrypoint-initdb.d/0000-compiled.sql
 
 RUN echo "Use LorisTest;" >> /docker-entrypoint-initdb.d/0001-paths.sql
 RUN echo "UPDATE Config SET Value='${BASE_DIR}/' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='base');" >> /docker-entrypoint-initdb.d/0001-paths.sql
 RUN echo "UPDATE Config SET Value='http://web:8000' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='url');" >> /docker-entrypoint-initdb.d/0001-paths.sql
+RUN echo "UPDATE users SET Password_hash=CONCAT('aa', MD5('aatestpass')), Pending_approval='N', Password_expiry='2100-01-01' WHERE ID=1;" >> /docker-entrypoint-initdb.d/0002-user.sql
```

Comment:
This doesn't work (or at least, it shouldn't). Password_hash stores the output of password_hash (which IIRC is bcrypt).